tests/nodeUnit.js

```diff
@@ -69,7 +69,45 @@ define(function(require) {
           'dist/public/index.html',
           'dist/public/manifest.json',
           'dist/public/offline-worker.js',
-          'dist/public/status.json'
+          'dist/public/status.json',
+          'dist/public/app-manifest.html',
```

Comment:
Having the right output is a good test result, but it either could check at
a different level or use the list of feature .md files as source of truth
and not a hard coded list that will be hard to maintain.

On Wednesday, January 13, 2016, Tim Abraldes <notifications@github.com>
wrote:

> In tests/nodeUnit.js
> <https://github.com/mozilla/platatus/pull/265#discussion_r49663819>:
>
> > @@ -69,7 +69,45 @@ define(function(require) {
> >            'dist/public/index.html',
> >            'dist/public/manifest.json',
> >            'dist/public/offline-worker.js',
> > -          'dist/public/status.json'
> > +          'dist/public/status.json',
> > +          'dist/public/app-manifest.html',
>
> It's a trade-off. Is the maintenance burden of the test worse than the
> chance that we might accidentally land a patch that causes some resources
> to 404?
>
> It's a pretty burdensome test, and when our app was a single page we might
> have gotten the same test coverage out of a quick glance at the output
> page. Now that we have a bunch of separate pages it seems like the test is
> more valuable than before.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/platatus/pull/265/files#r49663819>.
>


-- 
Web Craftsman | harald@mozilla.com | @digitarald [skype/irc/line] |
+1-650-963-6612 [wapp/telegram]